app/components/challenge-item.js

```diff
@@ -3,6 +3,28 @@ import _ from 'lodash/lodash';
 
 const { computed, inject } = Ember;
 
+function extValidate () {
+  if (this._hasError()) {
+    this.set('errorMessage', this._getErrorMessage());
+    return this.sendAction('onError', this.get('errorMessage'));
+  }
+  const value = this._getAnswerValue();
+  this.sendAction('onValidated', this.get('challenge'), this.get('assessment'), value);
+}
+
+function extSkip () {
+  this.set('errorMessage', null);
+  this.sendAction('onValidated', this.get('challenge'), this.get('assessment'), '#ABAND#')
+}
+
+let partialThrottle = (targetFunction) => {
```

Comment:
si je ne m'abuse, c'est parce que c'est une lambda, parce qu'on veut garder le `this` de l'instance idoine et non pas créer un nouveau `this`.
À tester idoinement, though.